pkg-r/vignettes/get-started.Rmd

```diff
@@ -0,0 +1,347 @@
+
+Model providers also typically offer a variety of models. To specify a particular model, use the `chat_*()` function's `model` argument. For example:
+
+```r
+chat_openai(model = "o3")
+```
+
+If you don't specify the `model` argument, the `chat_*()` function will use a reasonable default. For more information, see the individual `chat_*()` function's [documentation](https://ellmer.tidyverse.org/reference/index.html#chatbots). 
+
+<div class="callout">
+  <div class="title">Help me choose!</div>
+  <div class="body">
+If you're not sure which provider to choose, ellmer provides a [guide](https://ellmer.tidyverse.org/#providermodel-choice) to help you decide. 
+
+If you’re not ready to sign up for a cloud provider (e.g., OpenAI or Anthropic), you can run models locally (for free!) with [Ollama](https://ellmer.tidyverse.org/reference/chat_ollama.html).
+
+A local model is a good way to get started without incurring any cost or sharing your data with a cloud provider. You will need to [install Ollama](https://ellmer.tidyverse.org/reference/chat_ollama.html) and an open-source model (e.g., Llama 3.1). You can use the [`ollamar` package](https://hauselin.github.io/ollama-r/) to install particular models from R. 
+  </div>
+</div>
+
+### Set up credentials
+
+Next, authenticate with your LLM provider. Popular model providers like OpenAI and Anthropic require an API key. We recommend storing these API keys in your `.Renviron` (e.g., as `OPENAI_API_KEY` or `ANTHROPIC_API_KEY`).
+
+You can find the variable names in each `chat_*()` function's documentation (e.g., [`chat_openai()`](https://ellmer.tidyverse.org/reference/chat_openai.html#arg-api-key), [`chat_anthropic()`](https://ellmer.tidyverse.org/reference/chat_anthropic.html#arg-api-key)). There is additional information on [authentication](https://ellmer.tidyverse.org/#authentication) on the ellmer site.
+
+Once your credentials (if any) are in place, you're ready to get started. 
+
+## Create a basic chatbot
+
+Once you've identified which model provider you want to use and set up the necessary authentication, you're ready to create a chatbot. The following code creates a basic chatbot in a Shiny app. 
+
+Copy and paste the code into an R script, switching out `ellmer::chat_openai()` for your desired chat function. Save the file as `app.R` and then run the app.
+
+```r
+library(shiny)
+library(shinychat)
+
+ui <- bslib::page_fluid(
+  chat_ui("chat")
+)
+
+server <- function(input, output, session) {
+  chat <- ellmer::chat_openai()
+  
+  observeEvent(input$chat_user_input, {
+    stream <- chat$stream_async(input$chat_user_input)
+    chat_append("chat", stream)
+  })
+}
+
+shinyApp(ui, server)
+```
+
+Congrats, you now have a chat interface powered by an LLM of your choice! 🎉
+
+```{r, fig.cap="Screenshot of a conversation using shinychat.", fig.align='center', out.extra='class="rounded shadow"'}
```

Comment:
```suggestion
```{r, echo=FALSE, fig.cap="Screenshot of a conversation using shinychat.", fig.align='center', out.extra='class="rounded shadow"'}
```